README.md

```diff
@@ -5,7 +5,18 @@
 
 [![Release](https://github.com/braintree/braintree-ios-drop-in/actions/workflows/release.yml/badge.svg)](https://github.com/braintree/braintree-ios-drop-in/actions/workflows/release.yml)
 
-Welcome to Braintree's Drop-in SDK for iOS!
+## 📣 Announcements
+🚨🚨🚨 **The iOS Drop-In SDK is currently inactive and will be unsupported by November 2025** 🚨🚨🚨 
+
+As of December 2024, the iOS Drop-In SDK is inactive. No additional features will be added with the exception of security updates. Instead, implement a custom UI using the ![Braintree SDK (braintree_ios)](https://github.com/braintree/braintree_ios).
```

Comment:
The `!` in markdown is for images, so this would just show a broken image icon, instead we can use `[TEXT]()` for a standard link:
```suggestion
As of December 2024, the iOS Drop-In SDK is inactive. No additional features will be added with the exception of security updates. Instead, implement a custom UI using the [Braintree SDK (braintree_ios)](https://github.com/braintree/braintree_ios).
```